fmriprep/workflows/fieldmap/unwarp.py

```diff
@@ -24,14 +24,20 @@
 from nipype.pipeline import engine as pe
 from nipype.interfaces import utility as niu
 from nipype.interfaces import fsl
+from nipype.interfaces import afni
+from nipype.interfaces import freesurfer as fs
 from nipype.interfaces.ants import CreateJacobianDeterminantImage
 from niworkflows.interfaces.registration import ANTSApplyTransformsRPT, ANTSRegistrationRPT
-from niworkflows.interfaces.masks import ComputeEPIMask
+from niworkflows.interfaces.masks import BETRPT
 
 from fmriprep.interfaces import itk
 from fmriprep.interfaces import ReadSidecarJSON
 from fmriprep.interfaces.bids import DerivativesDataSink
 
+from nipype.interfaces import ants
+from fmriprep.interfaces import CopyHeader
+from fmriprep.interfaces.freesurfer import StructuralReference
```

Comment:
`StructuralReference` is also imported into `fmriprep.interfaces`, if you want to combine these lines.